### PR TITLE
feat(vscode): add quick generate skip code command

### PR DESCRIPTION
**Pull request by Simon-He95**

### CODE REVIEW

File: packages/vscode/src/index.ts [hunk 1]
```diff
@@ -37,12 +37,34 @@ export async function activate(ext: ExtensionContext) {
       ? toArray(root).map(r => path.resolve(projectPath, r))
       : [projectPath], config, status)
 
+  const skipMap = {
+    '<!-- @unocss-skip -->': ['<!-- @unocss-skip-start -->\n', '\n<!-- @unocss-skip-end -->'],
+    '// @unocss-skip': ['// @unocss-skip-start\n', '\n// @unocss-skip-end'],
```
Comment (zyyv) [inline packages/vscode/src/index.ts:42]: Can you also add `/* @unocss-skip-start */`? Thanks

Comment (Simon-He95) [reply packages/vscode/src/index.ts:42]: sure

**Comment by zyyv** [comment]
Can we add `keybindings` to mark the selected code snippet with magic-skip string? 
and automatically identify the comment types required by code files.

**Comment by Simon-He95** [comment]
> Can we add `keybindings` to mark the selected code snippet with magic-skip string?我们可以添加 `keybindings` 以使用 magic-skip 字符串标记所选代码片段吗？ and automatically identify the comment types required by code files.并自动识别代码文件所需的注释类型。

This is not easy to control. You may select a string, a variable, jsx, or a string jsx as a variable. I haven't figured out a good way to infer the selection of comments.

File: packages/vscode/src/index.ts [hunk 2]
```diff
@@ -37,12 +37,35 @@ export async function activate(ext: ExtensionContext) {
...
+  }
+
   ext.subscriptions.push(
     commands.registerCommand('unocss.reload', async () => {
       log.appendLine('🔁 Reloading...')
       await ctx.reload()
       log.appendLine('✅ Reloaded.')
     }),
+    commands.registerCommand('unocss.skip', async () => {
```
Comment (antfu) [inline packages/vscode/src/index.ts:52]:
```suggestion
    commands.registerCommand('unocss.insert-skip-annotation', async () => {
```

File: packages/vscode/package.json [hunk 3]
```diff
@@ -29,6 +29,11 @@
         "command": "unocss.reload",
         "title": "Reload UnoCSS",
         "category": "UnoCSS"
+      },
+      {
+        "command": "unocss.insert-skip-annotation",
+        "title": "Skip Code From UnoCSS",
```
Comment (antfu) [inline packages/vscode/package.json:35]:
```suggestion
        "title": "Insert `@unocss-skip` for the selection",
```

**Review by antfu — APPROVED** [review]

**Pull request merged**